frontend/dockerfile/instructions/parse_noparents.go

```diff
@@ -0,0 +1,6 @@
+//go:build !dfparents
+// +build !dfparents
+
+package instructions
+
+const parentsEnabled = false
```

Comment:
We actually don't need two build flag files for this.

See how it's done for [commands_runsecurity.go](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/instructions/commands_runsecurity.go#L27-L30) - we can use the `init()` function to set a global bool to `true` in `parse.go`.